apps/gamify/web-app/src/app/pages/users/profile/profile.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ProfileComponent } from './profile.component';
+
+describe('ProfileComponent', () => {
+  let component: ProfileComponent;
+  let fixture: ComponentFixture<ProfileComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ ProfileComponent ],
+      imports: [HttpClientModule],
+      providers: [HttpClient]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    // fixture = TestBed.createComponent(ProfileComponent);
+    // component = fixture.componentInstance;
+    // fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    // expect(component).toBeTruthy();
```

Comment:
Are you going to use this commented code?

Reply:
Probably not because since I'm not really making unit tests. Howevever it is a good starting point if someone wants to add unit tests to the component. But it is commented out because the unit test is not working and it would block the CI check in  Github (I am aware that this is not really a good practice)